### DISCUSSION TITLE
cost relevance question

### DISCUSSION THREAD

**Discussion by wanyingw193** [NREL/ReEDS-2.0#73] [Q&A: Analysis] [answered]
Hi,

Just want to confirm how ReEDS handles capital costs for existing plants if they are not fully amortized? Will these be calculated into the objective function? And how does ReEDS handle capital costs for newly installed power plants?

Also, are there any standard cost metrics that can be used to assess whether a scenario is reasonable?

Thank you so much!

**Comment by wesleyjcole** [comment, marked as answer]
In the objective function, only the capital costs for new assets are consider.  Existing assets are ignored as they are then sunk costs.  O&M costs are incurred each year until a plant is retired, so the model might choose to retire a plant early if O&M costs are great than the value provided by the plant.

There are several system cost outputs in the Bokeh viewer tool.  These system cost outputs include the option to include or exclude the capital costs of existing plants, and give you the option to amortize the capital costs of the plants.  The default HTML report includes the standard way we report system cost, but there is not really a "right" way to do it.  We have used many variations of system cost in our analysis work, which the ultimate choice of system cost metric driven by the analysis questions at hand.

**Comment by wanyingw193** [comment]
Hi! In the OF function, what is the data source for the market operating reserve cost file (cost_opres_market.csv)? In the 2021 Standard Program Report, it says that the cost of spinning reserves comes from "Spinning reserve cost equal to the heat rate reduction from operating at part load". What calculation formula is used? Why is the cost the same for all technologies? Why doesn't the cost vary by location, model year and the penetration of renewables? Thanks!

**Reply by wesleyjcole** [reply]
For more information on the operating reserves, include test scenarios to examine operating reserve impacts, see https://www.nrel.gov/docs/fy22osti/81706.pdf.

The market prices were the market prices from 2017 and based on https://www.nrel.gov/docs/fy19osti/72578.pdf.  We used that as a test case.  Otherwise operating reserve prices are endogenous in the model and vary based on location, year, and generation mix.

**Reply by wanyingw193** [reply]
Thank you!

I'm a little confused about the definitions between reserves. From https://www.nrel.gov/docs/fy19osti/72578.pdf, the price of flexible reserves is taken from "Table 8. Non-Spinning Contingency Reserve Requirements", and then at https://www.nrel.gov/docs/fy22osti/81706.pdf., flexible reserves are described as "Ramping Reserve"  outside of the 'Contingency Reserve' category

Then, I found another document in NREL (https://www.nrel.gov/docs/fy11osti/51978.pdf) which separates the contingency reserve and ramp reserve categories. Should I interpret the flexible Reserve as a combination of Contingency Reserve and Ramping Reserve? Thanks!

**Reply by wesleyjcole** [reply]
Operating reserve products vary from one market to another, so ReEDS names and definitions won't always directly map to market product names.  E.g., many markets to do not have a ramping product.

The summary is that ReEDS has three products: a fast, a medium, and a slow responding product.  They generally map to regulation reserves, spinning contigency reserves, and non-spinning/flexibility/ramping reserves.

**Reply by wanyingw193** [reply]
Thank you for the quick response!
